app/components/agent/Show.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Relay from 'react-relay/classic';
 import { Link } from 'react-router';
+import moment from 'moment';
```

Comment:
Should we just import just https://date-fns.org/v1.29.0/docs/differenceInSeconds instead of using moment? Seems to be the more minimal equivalent I seem people using, that also tree shakes nicely w/ webpack.

Reply:
I mean, if we didn’t already have moment loaded at all times I’d say sure :P